packages/i18next/src/lib/functions.ts

```diff
@@ -124,10 +146,11 @@ export async function sendLocalized<TKeys extends TFunctionKeys = string, TInter
  */
 export async function replyLocalized<TKeys extends TFunctionKeys = string>(target: Message, keys: TKeys | TKeys[]): Promise<Message>;
 /**
- * Replies to another message using an object of {@link LocalizedMessageOptions}.
+ * Replies to another message using an object of valid options.
```

Comment:
I'd say keep the links in, but append `object` instead. For instance, here,
```
* Replies to another message using a {@link LocalizedMessageOptions} object
```
Remember to use "an" instead of "a" if the object name starts with a vowel sound.

Reply:
Keep the links and also the `see` tsdoc?

Reply:
like this?
![image](https://user-images.githubusercontent.com/62367547/163721655-84446fd9-8d0d-4715-b351-003635db4dfa.png)
or this?
![image](https://user-images.githubusercontent.com/62367547/163721682-6d466b4f-38c6-489d-ad27-1a36d60919ea.png)

Reply:
Now I think about it, actually, the links should be directly in the parameter description. The description of this specific function should be something like "Replies to a message using localization" and the description should contain
```ts
* @param options A {@link LocalizedMessageOptions} object, requiring at least a `keys` field.
```

Reply:
Hmm I think this should be fine
![image](https://user-images.githubusercontent.com/62367547/163722215-5acefc33-968b-4581-895f-6de32614bc6c.png)

Reply:
hmm or
```ts
* Replies to another message using {@link LocalizedMessageOptions}.
```